src/style.css

```diff
@@ -1 +1,110 @@
-/* add styles here */
+@font-face {
+  font-family: Avenir;
+  src: url(fonts/Avenir-Book.ttf);
+  font-weight: 400;
+  font-style: normal;
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url(fonts/Avenir-Heavy.ttf);
+  font-weight: 900;
+  font-style: normal;
+}
+
+body {
+  margin-top: 20px;
+}
+
+/* #region form */
+
+.form {
+  display: flex;
+  position: relative;
+  flex-direction: column;
+  gap: 20px;
+}
+
+.form__bar {
+  /* element settings */
+  box-sizing: border-box;
+  border: 1px solid #e1e7ed;
+  border-radius: 4px;
+  font-family: Avenir, Helvetica, sans-serif;
+
+  /* content ssetings */
+  color: #3d4e61;
+  font-weight: 400;
+}
+
+.form__bar--big {
+  height: 70px;
+  padding-left: 62px;
+  font-size: 16px;
+  box-shadow: 0 1px 8px 0 #3d4e611a;
+}
+
+.form__bar--small {
```

Comment:
Don't fix container size (if there is no such a requirement). As with the previous comment, consider using padding instead of a fixed height to allow the content to dictate the size of the input field.